packages/create-request-form/src/lib/invoice/form.svelte

```diff
@@ -0,0 +1,358 @@
+<script lang="ts">
+  import {
+    currencies,
+    formatDate,
+    inputDateFormat,
+    calculateItemTotal,
+  } from "$src/utils";
+  import {
+    Input,
+    Button,
+    Labels,
+    Dropdown,
+    Accordion,
+  } from "@requestnetwork/shared";
+
+  export const invoiceNumber: number = 1;
+  let creatorId = "";
+  export let formData: CustomFormData;
+  export let handleCurrencyChange: (value: string) => void;
+
+  $: {
+    creatorId = formData.creatorId;
+  }
+
+  const calculateInputWidth = (value: string) => {
+    const baseWidth = 20;
+    const width = Math.max(value?.length, baseWidth) * 7.6;
+    return `${width}px;`;
+  };
+
+  const handleAdditionalInfo = (event: Event) => {
+    const target = event.target as HTMLInputElement;
+    const { id, value } = target;
+    const fieldType = id.split("-")[0];
+    const fieldName = id.split("-")[1];
+
+    (formData as any)[fieldType][fieldName] = value;
+  };
+
+  const handleInput = (event: Event, itemIndex?: number) => {
+    const target = event.target as HTMLInputElement;
+    const { id, value } = target;
+    const fieldName = id.split("-")[0];
+
+    if (typeof itemIndex === "number") {
+      (formData.items[itemIndex] as any)[fieldName] = value;
+    } else {
+      if (id in formData) {
+        (formData as any)[id] = value;
+      }
+    }
+  };
+
+  const addInvoiceItem = () => {
+    const newItem = {
+      description: "",
+      quantity: 1,
+      unitPrice: 0,
+      discount: 0,
+      tax: 0,
+      amount: "",
+    };
+    formData.items = [...formData.items, newItem];
+  };
+
+  const removeInvoiceItem = (index: number) => {
+    formData.items = formData.items.filter((_, i) => i !== index);
+  };
+</script>
+
+<form
+  class="h-fit bg-white flex flex-col w-[120%] p-[20px] shadow-small gap-[20px]"
+>
+  <div class="flex items-start w-full">
+    <div class="flex items-center gap-[12px]">
+      <h2 class="text-dark-blue font-bold text-[20px] w-full">Invoice #</h2>
+      <Input
+        className="w-full"
+        id="invoiceNumber"
+        type="text"
+        value={formData.invoiceNumber}
+        {handleInput}
+        style={`width: ${calculateInputWidth(formData.invoiceNumber.toString())}`}
+      />
+    </div>
+    <div class="flex flex-col gap-[9px] ml-auto w-fit">
+      <p>Issued on {formData.issuedOn && formatDate(formData.issuedOn)}</p>
+      <p>
+        Payment due by {formData.dueDate && formatDate(formData.dueDate)}
+      </p>
+    </div>
+  </div>
+  <div class="flex gap-[20px]">
+    <div class="flex flex-col w-full max-w-[500px] gap-[20px]">
```

Comment:
Please rearrange the form sections to match [this Figma](https://www.figma.com/file/cEkRPhER2ArCnOwNVDmVVA/Request?type=design&node-id=164-299&mode=design&t=d5G7FimIxA9YQ59j-0)